index.yaml

```diff
@@ -1,15 +1,5 @@
 indexes:
 
-# AUTOGENERATED
```

Comment:
Should we keep this note? It is still generated by a script right?

Reply:
Not sure about that, 
@vojtechjelinek what do you say?

Reply:
I think we can keep it.